frontend/src/lib/stores.js

```diff
@@ -1,27 +1,41 @@
 import { writable } from 'svelte/store'
 import languages from '../languages/languages.json'
 
-export const currentLanguage = writable(languages[0])
-let _currentLanguage
-currentLanguage.subscribe((value) => (_currentLanguage = value))
+let words = [
```

Comment:
This doesn't look great but it breaks if we don't initialize words.

Reply:
Yeah not like this.

You'll need something like

```js
let wordCache
let cachedLanguage
```

and then

```js
async function loadWords(language) {
  if (cachedLanguage !== language)  {
    wordCache = await ...
    cachedLanguage = language
  }
  return wordCache
}
```

Reply:
though I'd change the "loadWords" name to "getWords"

Reply:
oh yeah, you don't need `language` argument as you can access `_currentLanguage`